### PR TITLE
[zh] Localize /feature-gates/b*.md

### PR DESCRIPTION
**Pull request by Fan-Lin**
Related to issue #44410 . Localize b*.md files:
1. balance-attached-node-volumes
2. block-volume
3. bound-service-account-token-volume

### CODE REVIEW

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/balance-attached-node-volumes.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  render: false
+---
+
+<!--
+Include volume count on node to be considered for
+balanced resource allocation while scheduling. A node which has closer CPU,
+memory utilization, and volume count is favored by the scheduler while making decisions.
+-->
+在进行平衡资源分配的调度时，考虑节点上的卷数。
+调度器在决策时会优先考虑 CPU、内存利用率和卷数更近的节点。
```
Comment (chenrui333) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/balance-attached-node-volumes.md:17]:
```suggestion
在调度时考虑节点上的卷数量，以实现平衡资源分配。
调度器在决策时会优先考虑 CPU、内存利用率和卷数更近的节点。
```

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/block-volume.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  render: false
+---
+
+<!--
+Enable the definition and consumption of raw block devices in Pods.
+See [Raw Block Volume Support](/docs/concepts/storage/persistent-volumes/#raw-block-volume-support)
+for more details.
+-->
+在 Pod 中启用原始块设备的定义和使用。有关更多详细信息，
+请参见[原始块卷支持](/zh-cn/docs/concepts/storage/persistent-volumes/#raw-block-volume-support)。
```
Comment (chenrui333) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/block-volume.md:17]:
```suggestion
启用在 Pods 中定义和使用原始块设备。
更多详情请参见[原始块卷支持](/zh-cn/docs/concepts/storage/persistent-volumes/#raw-block-volume-support)。
```

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/bound-service-account-token-volume.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+Check [Bound Service Account Tokens](https://github.com/kubernetes/enhancements/blob/master/keps/sig-auth/1205-bound-service-account-tokens/README.md)
+for more details.
+-->
+迁移 ServiceAccount 卷以使用由
+ServiceAccountTokenVolumeProjection 组成的投射卷。集群管理员可以使用
+`serviceaccount_stale_tokens_total` 度量值来监控依赖于扩展令牌的负载。
+如果没有这种类型的负载，你可以在启动 `kube-apiserver` 时添加
+`--service-account-extend-token-expiration=false` 参数关闭扩展令牌。
+
+查看[绑定服务账号令牌](https://github.com/kubernetes/enhancements/blob/master/keps/sig-auth/1205-bound-service-account-tokens/README.md)获取更多详细信息。
```
Comment (chenrui333) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/bound-service-account-token-volume.md:27]:
```suggestion
将 ServiceAccount 卷迁移到使用由
ServiceAccountTokenVolumeProjection 组成的投射卷。集群管理员可以使用
`serviceaccount_stale_tokens_total` 指标来监控依赖于扩展令牌的工作负载。
如果没有这样的工作负载，你可以在启动 `kube-apiserver` 时添加
`--service-account-extend-token-expiration=false` 标志来关闭扩展令牌。

查更多详情请参见[绑定服务账号令牌](https://github.com/kubernetes/enhancements/blob/master/keps/sig-auth/1205-bound-service-account-tokens/README.md)。
```

**Review by chenrui333 — COMMENTED** [review]
left couple of comments

/assign

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/block-volume.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  list: never
+  render: false
+---
+
+<!--
+Enable the definition and consumption of raw block devices in Pods.
+See [Raw Block Volume Support](/docs/concepts/storage/persistent-volumes/#raw-block-volume-support)
+for more details.
+-->
+启用在 Pods 中定义和使用原始块设备。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/block-volume.md:16]:
```suggestion
启用在 Pod 中定义和使用原始块设备。
```
翻译成中文后，复数 s 可以去掉

**Review by windsonsea — COMMENTED** [review]
pick a nit.

/approve

**Pull request merged**